webapp/src/pages/eventConfig/components/EventConfigComponent.js

```diff
@@ -137,275 +370,378 @@ class EventConfigComponent extends Component {
-
-              <label className={"col-sm-2 col-form-label"} htmlFor="end_date">
-                {t("End Date")}
-              </label>
-
-              <div className="col-sm-4">
-                <DateTimePicker
-                  clearIcon={null}
-                  disableClock={true}
-                  onChange={e => this.updateDateTimeEventDetails("end_date", e)}
-                  value={new Date(updatedEvent.end_date)} />
+                  name="url"
+                  type="text"
+                  placeholder={t("Event website (e.g. www.deeplearningindaba.com)")}
+                  value={updatedEvent.url}
+                  required={true}
+                  onChange={e => this.updateEventDetails("url", e)}
+                  />
               </div>
             </div>
 
             <div className={"form-group row"}>
               <label
                 className={"col-sm-2 col-form-label"}
                 htmlFor="application_open">
+                {<span className="required-indicator">*</span>}
                 {t("Application Open")}
               </label>
 
               <div className="col-sm-4">
-                <DateTimePicker
-                  clearIcon={null}
-                  disableClock={true}
+                <FormDate
+                  id="application_open"
+                  name="application_open"
+                  value={updatedEvent.application_open}
+                  required={true}
                   onChange={e =>
-                    this.updateDateTimeEventDetails("application_open", e)}
-                  value={new Date(updatedEvent.application_open)} />
+                    this.updateDateTimeEventDetails("application_open", e)}/>
               </div>
 
               <label
                 className={"col-sm-2 col-form-label"}
-                htmlFor="application_close"
-              >
+                htmlFor="application_close">
+                {<span className="required-indicator">*</span>}
                 {t("Application Close")}
               </label>
 
               <div className="col-sm-4">
-                <DateTimePicker
-                  clearIcon={null}
-                  disableClock={true}
+                <FormDate
+                  id="application_close"
+                  name="application_close"
+                  value={updatedEvent.application_close}
+                  required={true}
                   onChange={e =>
-                    this.updateDateTimeEventDetails("application_close", e)
-                  }
-                  value={new Date(updatedEvent.application_close)} />
+                    this.updateDateTimeEventDetails("application_close", e, close_time)
+                  } />
               </div>
 
             </div>
+
             <div className={"form-group row"}>
               <label
                 className={"col-sm-2 col-form-label"}
                 htmlFor="review_open">
+                {<span className="required-indicator">*</span>}
```

Comment:
Is this working now? I think i remember you leaving a comment saying it wasn't working, but I can't find it... (Github's PR commenting interface is confusing!)

Reply:
Also you don't need the squigly brackets around the span :)

Reply:
Yes, I worked out how to do it :)